### ISSUE TITLE
UIRouter option could accept a string

### ISSUE THREAD

**Issue by forwardadvance** [CanopyTax/single-spa-angular1#32]
Nick Johnson from Beamery here, halfway through the integration, I have a PR for you.

We always log our PRs against tickets, so here's a ticket, PR to follow.

It would be useful to be able to write:

```
const ng1Lifecycles = singleSpaAngular1({
  angular: angular,
  domElementGetter: () => document.getElementById('main-content'),
  mainAngularModule: 'app',
  uiRouter:'core',
	preserveGlobal: false,
	template: '<my-component />',
});
```

to output:

```
<div ui-view="core"></div>
```

**Comment by forwardadvance** [comment]
PR is here: https://github.com/CanopyTax/single-spa-angular1/pull/33

**Comment by forwardadvance** [comment]
This addresses a specific issue we had while trying to migrate our legacy app - it had a named insertion point.

**Comment by joeldenning** [comment]
Merged and released as single-spa-angular1@2.6.0
